doc/api/https.md

```diff
@@ -7,47 +7,53 @@ separate module.
 
 ## Class: https.Agent
 <!-- YAML
-added: v0.4.5
+added: v0.4.4
 -->
 
-An Agent object for HTTPS similar to [`http.Agent`][].  See [`https.request()`][]
-for more information.
+**Inherits:** [`http.Agent`]
+
+An Agent object for HTTPS.
 
 ## Class: https.Server
 <!-- YAML
 added: v0.3.4
 -->
 
-This class is a subclass of `tls.Server` and emits events same as
-[`http.Server`][]. See [`http.Server`][] for more information.
+**Inherits:** [`tls.Server`]
+**Implements:** [`http.Server`]
```

Comment:
I'm open to suggestions on naming, formatting, etc. The goal here is to document inheritance and the like, which also helps reduce redundant documentation.

I just thought of something that would be neat, but is out of the scope of what I'm trying to do at the moment: if we could (dynamically) integrate collapsed (by default) sections that would show inherited events, methods, and properties (one collapsed section per type). That would help prevent having to navigate to a separate page to see inherited API while still leaving the UI less cluttered and easy to see API unique to that object.